httpserver/http_connection.h

```diff
@@ -123,6 +123,10 @@ class connection_t : public std::enable_shared_from_this<connection_t> {
 
     ChannelEncryption<std::string>& channel_cipher_;
 
+    // The timer for repeating an action within one connection
+    boost::asio::steady_timer repeat_timer_;
```

Comment:
I was actually thinking it could be used in other types of requests (not at the same time of course). It would be silly to have separate timers for each request and only use one of them depending on the type of request.